com.unity.render-pipelines.universal/Shaders/SpatialMappingOcclusion.shader

```diff
@@ -0,0 +1,72 @@
+Shader "Universal Render Pipeline/SpatialMappingOcclusion"
```

Comment:
change to `Universal Render Pipeline/VR/SpatialMapping/Occlusion` for better consistency with builtin